Cargo.toml

```diff
@@ -157,6 +157,7 @@ sqllogictest = "0.17"
 strum = { version = "0.25.0", features = ["derive"] }
 syn = { version = "2", features = ["full", "extra-traits"] }
 syntect = { version = "5.0.0", default-features = false, features = ["default-fancy"] }
+sys-info = "0.9.1"
```

Comment:
I'm pretty sure this one is going to be fine because it's a pretty trivial dependency but worth mentioning:

Any time we land something that has an OS dependency (like this does), I'd like the person landing it to be a little heads up in case there's a problem on whichever of linux/macos/windows they don't run on.